code/cavity2d/CAVITY_2D_STAGGERED.f90

```diff
@@ -1,507 +1,507 @@
-	  W(3,MIP,J)=-W(3,MI,J)
-	  ENDDO
-!J=MJP
-      DO I=0,MIP
-	  W(1,I,MJP)=W(1,I,MJ)
-	  W(2,I,MJP)=UIN
-	  W(3,I,MJ)=VIN
-	  ENDDO
-!J=0
-	  DO I=0,MIP
-	  W(1,I,0)=W(1,I,1)
-	  W(2,I,0)=-W(2,I,1)
-	  W(3,I,0)=0.
-	  ENDDO
-!I=0
-	  RETURN
-	  END
-!-----------------------------------------------------
-      SUBROUTINE SYUU(DCT,APT,AMT,BPT,BMT,CC,UT)
-	  USE PISO
-	  REAL DCT(-2:MI+2,-2:MJ+2),APT(-2:MI+2,-2:MJ+2),AMT(-2:MI+2,-2:MJ+2)
-	  REAL BPT(-2:MI+2,-2:MJ+2),BMT(-2:MI+2,-2:MJ+2)
-	  REAL CC(-2:MI+2,-2:MJ+2),UT(-2:MI+2,-2:MJ+2),UTO(-2:MI+2,-2:MJ+2)
-
-      MIM=MI-1; MJM=MJ-1
-
-	  ITT=0
-1000  ITT=ITT+1
-
-      DO I=1,MIM; DO J=1,MJ
-	  UTO(I,J)=UT(I,J)
-	  ENDDO; ENDDO
-
-      DO J=1,MJ; DO I=1,MIM
-	  IP=I+1; IM=I-1; JP=J+1; JM=J-1
-	  TEMP=CC(I,J)-APT(I,J)*UT(IP,J)-AMT(I,J)*UT(IM,J)&
-	               &-BPT(I,J)*UT(I,JP)-BMT(I,J)*UT(I,JM)
-
-      UT(I,J)=TEMP/DCT(I,J)
-	  ENDDO; ENDDO
-
-
-
-	  ERR=0.
-	  DO J=1,MJ; DO I=1,MIM
-	  UTEMP=UT(I,J)-UTO(I,J)
-	  ERR=ERR+UTEMP*UTEMP
-	  ENDDO; ENDDO
-	  ERR=SQRT(ERR/(MIM*MJ))
-
-
-	  IF(ERR .GT. 0.000001 .OR. ITT .LT. 10) GOTO 1000
-
-      RETURN
-	  END
-
-!=================================================================
-SUBROUTINE SYUV(DCT,APT,AMT,BPT,BMT,CC,UT)
-	  USE PISO
-	  REAL DCT(-2:MI+2,-2:MJ+2),APT(-2:MI+2,-2:MJ+2),AMT(-2:MI+2,-2:MJ+2)
-	  REAL BPT(-2:MI+2,-2:MJ+2),BMT(-2:MI+2,-2:MJ+2)
-	  REAL CC(-2:MI+2,-2:MJ+2),UT(-2:MI+2,-2:MJ+2),UTO(-2:MI+2,-2:MJ+2)
-
-      MIM=MI-1; MJM=MJ-1
-
-	  ITT=0
-1000  ITT=ITT+1
-
-      DO I=1,MI; DO J=1,MJM
-	  UTO(I,J)=UT(I,J)
-	  ENDDO; ENDDO
-
-      DO J=1,MJM; DO I=1,MI
-	  IP=I+1; IM=I-1; JP=J+1; JM=J-1
-	  TEMP=CC(I,J)-APT(I,J)*UT(IP,J)-AMT(I,J)*UT(IM,J)&
-	               &-BPT(I,J)*UT(I,JP)-BMT(I,J)*UT(I,JM)
-
-      UT(I,J)=TEMP/DCT(I,J)
-	  ENDDO; ENDDO
-
-	  ERR=0.
-	  DO J=1,MJM; DO I=1,MI
-	  UTEMP=UT(I,J)-UTO(I,J)
-	  ERR=ERR+UTEMP*UTEMP
-	  ENDDO; ENDDO
-	  ERR=SQRT(ERR/(MI*MJM))
-
-
-	  IF(ERR .GT. 0.000001 .OR. ITT .LT. 10) GOTO 1000
-
-      RETURN
-	  END
-!------------------------------------------------------------
-      SUBROUTINE SYUP(DCT,APT,AMT,BPT,BMT,CC,UT)
-	  USE PISO
-	  REAL DCT(-2:MI+2,-2:MJ+2),APT(-2:MI+2,-2:MJ+2),AMT(-2:MI+2,-2:MJ+2)
-	  REAL BPT(-2:MI+2,-2:MJ+2),BMT(-2:MI+2,-2:MJ+2)
-	  REAL CC(-2:MI+2,-2:MJ+2),UT(-2:MI+2,-2:MJ+2),UTO(-2:MI+2,-2:MJ+2)
-
-	  ITT=0; IGT=MI*MJ
-1000  ITT=ITT+1
-
-      DO I=1,MI; DO J=1,MJ
-	  UTO(I,J)=UT(I,J)
-	  ENDDO; ENDDO
-
-      DO I=1,MI; DO J=1,MJ
-	  IP=I+1; IM=I-1; JP=J+1; JM=J-1
-	  TEMP=CC(I,J)-APT(I,J)*UT(IP,J)-AMT(I,J)*UT(IM,J)&
-	               &-BPT(I,J)*UT(I,JP)-BMT(I,J)*UT(I,JM)
-      UT(I,J)=TEMP/DCT(I,J)
-	  ENDDO; ENDDO
-
-
-	  ERR=0.
-	  DO J=1,MJ ; DO I=1,MI
-	  UTEMP=UT(I,J)-UTO(I,J)
-	  ERR=ERR+UTEMP*UTEMP
-	  ENDDO; ENDDO
-	  ERR=SQRT(ERR/(MI*MJ))
-
-
-	  IF(ERR .GT. 0.000001 .OR. ITT .LT. 10) GOTO 1000
-!	  WRITE(*,*)'GS_ITER',ITT,ERR
-      RETURN
-	  END
+!==========================2D CAVITY FLOW=========================================  

```

Comment:
You are changing this back to dos format again.  Please don’t do this.  You need to configure your editor to do the right thing.

```
$ file CAVITY_2D_STAGGERED.f90
CAVITY_2D_STAGGERED.f90: ASCII text, with CRLF line terminators
```